src/network/network.cpp

```diff
@@ -619,7 +619,9 @@ bool Network::BeginClient(const char* host, unsigned short port)
 	server_address.Resolve(host, port);
 	status = NETWORK_STATUS_RESOLVING;
 
-	window_network_status_open("Resolving...");
+	char str_resolving[13];
```

Comment:
Good suggestion. I found other instances where an array size of 256 was used so I implemented that.